### PR TITLE
docs: add oldest unacked message age metric documentation update for GCP Pub/Sub Scaler

### PR DESCRIPTION
**Pull request by Friedrich42**
### Checklist

- [x] Commits are signed with Developer Certificate of Origin (DCO)

Fixes https://github.com/kedacore/keda/issues/2226

### CODE REVIEW

**Comment by Friedrich42** [comment]
https://deploy-preview-579--vigorous-franklin-8edd65.netlify.app/docs/2.5/scalers/gcp-pub-sub/

**Review by zroubalik — APPROVED** [review]

**Comment by tomkerkhove** [comment]
As per the issue, we should at least be backward compatible and deprecate the old field which is reflected in the docs

File: content/docs/2.5/scalers/gcp-pub-sub.md [hunk 1]
```diff
@@ -15,16 +15,17 @@ This specification describes the `gcp-pubsub` trigger for Google Cloud Platform
 triggers:
 - type: gcp-pubsub
   metadata:
-    subscriptionSize: "5" # Optional - Default is 5
+    mode: "SubscriptionSize" # Optional - Default is SubscriptionSize - SubscriptionSize or OldestUnackedMessageAge
+    value: "5" # Optional - Default is 5 for SubscriptionSize | Default is 10 for OldestUnackedMessageAge
```
Comment (tomkerkhove) [inline content/docs/2.5/scalers/gcp-pub-sub.md:19]: Need to document the backwards compatibility so cannot remove this

**Review by tomkerkhove — CHANGES_REQUESTED** [review]
Need to document the backward compatibility so cannot remove the old config, yet.

File: content/docs/2.5/scalers/gcp-pub-sub.md (hunk 1 above)
Comment (zroubalik) [reply content/docs/2.5/scalers/gcp-pub-sub.md:19]: Good catch.

**Review by tomkerkhove — APPROVED** [review]
Thanks a ton!

**Pull request merged**